tests/test_distance_map_average.py

```diff
@@ -0,0 +1,24 @@
+from minisom import MiniSom
+from sklearn.datasets import load_iris
+from matplotlib import pyplot as plt
+
+
+def test_distance_map():
+    """
+    Compare distance maps with averaged or added neighbour distances.
```

Comment:
all the tests of MiniSom are in `minisom.py` and there's already a test for `distance_map`: https://github.com/JustGlowing/minisom/blob/master/minisom.py#L753